sdk/storage/azure-storage-queue/tests/test_queue.py

```diff
@@ -32,25 +35,22 @@
 
 from queuetestcase import (
     QueueTestCase,
-    TestMode,
-    record,
     LogCaptured,
 )
 
 # ------------------------------------------------------------------------------
 TEST_QUEUE_PREFIX = 'pythonqueue'
 
+FAKE_STORAGE = FakeStorageAccount(
+    name='pyacrstorage',
+    id='')
 
 # ------------------------------------------------------------------------------
 
 
 class StorageQueueTest(QueueTestCase):
     def setUp(self):
         super(StorageQueueTest, self).setUp()
-
-        queue_url = self._get_queue_url()
-        credentials = self._get_shared_key_credential()
-        self.qsc = QueueServiceClient(account_url=queue_url, credential=credentials)
         self.test_queues = []
 
     def tearDown(self):
```

Comment:
Now that we're no longer using unittest.Testcase - I don't think the self.setUp and self.tearDown are being used any more - all this code should now be incorporated in the preparers, or cleaned up in some other way.
If we are creating extra queues that need to be deleted - we can probably call this function explicitly in the test where needed.
```python
try:
    # test code
finally:
    self._clean_up()
```

Reply:
I just occurred to me that we're cleaning up the resource group afterwards - so left over queues shouldn't be a problem :)

Reply:
we are not using them anymore. i've removed setup and teardown